bundles/sirix-core/src/main/java/org/sirix/utils/IntToObjectMap.java

```diff
@@ -329,8 +329,7 @@ public boolean containsKey(int key) {
    *         false otherwise.
    */
   public boolean containsValue(Object o) {
-    for (Iterator<T> iterator = iterator(); iterator.hasNext(); ) {
-      T object = iterator.next();
+    for (T object : this) {
```

Comment:
can explain why you used enhanced for-loop in this situation?

Reply:
No other reason than just making it more compact

Reply:
Though I can see it being an issue for some people if I'm being honest

Reply:
I think it's great :-)

Reply:
thank you